README.md

```diff
@@ -50,8 +50,8 @@ Overall, *NEAR* provides a wide range of tools for developers to easily build ap
 [as-sdk]: https://github.com/near/near-sdk-as
 [examples-url]: https://near.dev
 [docs-url]: http://docs.nearprotocol.com
```

Comment:
Looks like this as well should be updated.

Reply:
Yeah, I've updated it.